po/es.po

```diff
@@ -3057,40 +3054,39 @@ msgid ""
 "the debug output. We could also have used `{a}` and `{a:?}` without "
 "specifying the value after the format string."
 msgstr ""
-"En la función main, la palabra print pregunta por la implementación de debug "
-"con `?` para el formato de parámetro: `{}` que da una salida estándar, `{:?}"
-"`. También podemos usar `{a}` y \\`{a:?} sin especificar los valores luego "
-"del formato string."
+"En la función principal, la instrucción de impresión solicita la "
```

Comment:
"main" is the proper noun and name of the main() function, better leave it untranslated.